docs/parameters.rst

```diff
@@ -2,3 +2,276 @@
+- ``Radiative Forcing|HFC143a`` (W/m\*\*2)
+- ``Radiative Forcing|HFC152a`` (W/m\*\*2)
+- ``Radiative Forcing|HFC227ea`` (W/m\*\*2)
+- ``Radiative Forcing|HFC236fa`` (W/m\*\*2)
+- ``Radiative Forcing|HFC245fa`` (W/m\*\*2)
+- ``Radiative Forcing|HFC365mfc`` (W/m\*\*2)
+- ``Radiative Forcing|CFC11`` (W/m\*\*2)
+- ``Radiative Forcing|CFC12`` (W/m\*\*2)
+- ``Radiative Forcing|CFC113`` (W/m\*\*2)
+- ``Radiative Forcing|CFC114`` (W/m\*\*2)
+- ``Radiative Forcing|CFC115`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC22`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC141b`` (W/m\*\*2)
+- ``Radiative Forcing|HCFC142b`` (W/m\*\*2)
+- ``Radiative Forcing|SF6`` (W/m\*\*2)
+- ``Radiative Forcing|SO2F2`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1202`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1211`` (W/m\*\*2)
+- ``Radiative Forcing|Halon1301`` (W/m\*\*2)
+- ``Radiative Forcing|Halon2402`` (W/m\*\*2)
+
+
+Material Fluxes
+***************
+
+These variables can be used to store the flux of material within the model.
+They should be of the form ``X to Y Flux`` where the material is flowing from ``X`` into ``Y`` (and hence negative values represent flows from ``Y`` into ``X``).
+OpenSCM models should understand all of the following variables.
+
+
+- ``Land to Air Flux|CO2|Permafrost`` (mass carbon / time) - land to air flux of |CO2| from permafrost
+- ``Land to Air Flux|CH4|Permafrost`` (mass methane / time)
+
+
+Other
+*****
+
+Other variables which should be recognised by OpenSCM adapters are given below.
+
+- ``Surface Temperature`` (K) - surface air temperature i.e. tas
+- ``Ocean Temperature`` (K) - surface ocean temperature i.e. tos
+- ``Ocean Heat Content`` (J)
+- ``Sea Level Rise`` (mm)
+
+
+Regions
+-------
+
+Similarly to variables, regions are also provided in a hierarchy separated by the ``|`` character.
+To date, regions which are higher in the hierarchy are the sum of all the regions which are one level below them in the hierarchy (be careful of this when looking at e.g. |CO2| concentration data at a regional level).
+
+All OpenSCM adapaters must understand the following regions:
+
+- ``World``
+- ``World|Northern Hemisphere``
+- ``World|Northern Hemisphere|Ocean``
+- ``World|Northern Hemisphere|Land``
+- ``World|Southern Hemisphere``
+- ``World|Southern Hemisphere|Ocean``
+- ``World|Southern Hemisphere|Land``
+- ``World|Ocean``
+- ``World|Land``
+- ``World|R5ASIA``
+- ``World|R5REF``
+- ``World|R5MAF``
+- ``World|R5OECD``
+- ``World|R5LAM``
+- ``World|R5.2ASIA``
+- ``World|R5.2REF``
+- ``World|R5.2MAF``
+- ``World|R5.2OECD``
+- ``World|R5.2LAM``
+- ``World|Bunkers``
+
+
+Configuration
+-------------
+
+Each model will have its own set of configuration parameters and conventions.
+In OpenSCM we allow the user to pass these to and from the model via the adapter, following the model's own internal conventions for naming.
+However, we also insist that models understand the following configuration options.
+
+- ``ecs`` (K) - equilibrium climate sensitivity
```

Comment:
Shouldn't we spell these out?

Reply:
how do you mean spell them out?

Reply:
we removed this statement anyway so all good